CHANGES/6725.bugfix

```diff
@@ -0,0 +1 @@
+Documented system requirements for ansible when using the installer.
```

Comment:
I believe `.doc` is better than `.bugfix` in this case

Reply:
You might be right!